### DISCUSSION TITLE
Welcome to Anvil Extras Discussions!

### DISCUSSION THREAD

**Discussion by meatballs** [anvilistas/anvil-extras#217] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rhurlbatt** [comment]
Well this is a whole new section I did not even know existed!!

Hello, my name is Rick and I am an Anvil-aholic.

This would be my first time contributing to a project and I am realising just how little I actually know but I am definitely keen to get my hands dirty!

This project has helped my own projects immensely so much love from me :)

**Comment by meatballs** [comment]
> Well this is a whole new section I did not even know existed!!

Well, it didn't exist until earlier today!

**Comment by ralixit** [comment]
I'm Rafael. Hi 👋👋 all, I'm so grateful I stumbled upon this community.
Presently I'm new to Anvil, coming from just learning Django as I was able to publish a site. 

I came across Anvil. I knew immediately it was the way to go.
Right now, I'm tackling the anvil_extras piece, especially authentication & authorization, And I am having trouble with it.

 Using the authentication, I could only get it to work by passing in the function name into the decorator; as far as authorizing, I'm still trying to understand, but it should be simple, but I'm having trouble.
